api/alias_api/__init__.py

```diff
@@ -72,24 +136,42 @@ def import_alias_api():
             "{}.pyd".format(module_name),
         )
     )
-
     if not os.path.exists(module_path):
-        return
+        raise AliasPythonAPIImportError("Module does not exist {}".format(module_path))
 
+    # Find and create the module spec object for the Alias Python API
     spec = importlib.util.spec_from_file_location(module_name, module_path)
-    alias_api = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(alias_api)
+    if not spec:
+        # NOTE importlib.util.spec_from_location does not seem to find the module on macos
+        raise AliasPythonAPIImportError(
+            "Could not find the Alias Python API module {}".format(module_path)
+        )
+
+    try:
+        alias_api = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(alias_api)
+    except Exception as e:
+        info_msg = "The specific Alias version that the Alias Python API requires must be installed. The Alias bin directory path must be in the system environment PATH variable. If more than one version of Alias is installed, it must appear before all other versions in the PATH."
+        raise AliasPythonAPIImportError(
+            "{error}\n{info}".format(error=str(e), info=info_msg)
+        )
 
-    # add the newly created module oject to sys.modules and remap the globals accessor to point at our new module
+    # Add the newly created module oject to sys.modules and remap the globals accessor to point at our new module
     sys.modules["alias_api"] = alias_api
     globals()["alias_api"] = sys.modules["alias_api"]
 
 
+#
+# First check the python version is at least 3
+#
```

Comment:
While we are here, should we currently restrict this to be just 3.7 since we know others had issues trying 3.9?

Reply:
@rob-aitchison Do we know what were the issues for 3.9?

Reply:
@staceyoue no. Let's leave it as-is for now and re-consider if we get more specifics about it - thanks!